pkg/action/action.go

```diff
@@ -53,7 +53,6 @@ type FrizbeeAction struct {
 	ImagesReplacer  *replacer.Replacer
 	BFS             billy.Filesystem
 	Repo            *git.Repository
-	bodyBuilder     *strings.Builder
```

Comment:
Removing `bodyBuilder` seems like a (nice) simple refactor which doesn't change behavior.  I could approve that change real fast if it wasn't entangled in the behavior change around `ActionsPaths` changing contents.

Reply:
https://github.com/stacklok/frizbee-action/pull/54